client/my-sites/themes/themes-magic-search-card/style.scss

```diff
@@ -28,7 +28,7 @@
 		flex: 0 1 auto;
 		margin: 0 5px 0 10px;
 		height: 36px;
-		border: solid 1px var( --color-border-subtle );
+		border: solid 1px var( --color-neutral-10 );
```

Comment:
One benefit of using `--color-border-subtle` is that it can change depending on the dashboard color scheme, whereas `--color-neutral-10` will not. Not a big issue in the current state where I think it's the same throughout, but if we wanted to introduce a dark theme or change the colors in the main content area, this would have to be overwritten.

If it's possible to "future proof" by going the other way and using `--color-border-subtle` for the filters so they still match, I'd do that.

Reply:
It **does** sound better to use `--color-border-subtle` but the reason I went with `--color-neutral-10` is that the `Info` button right above the filter already use the neutral variant, so I thought that it made more sense.

![image](https://user-images.githubusercontent.com/375980/127039590-8eb0869a-725e-4549-9e54-79ee2e6a5215.png)

 How do you feel about that?

Reply:
Yeah, let's go with `--color-neutral-10`. We also have a `--color-border` but that maps to `--color-neutral-20`. None of the default button components use the mapped color value, so 🤷‍♀️